includes/api/episodes.php

```diff
@@ -66,11 +77,49 @@ function episodes_api($request)
         'files' => \podlove_pwp5_files($episode, null),
         'content' => apply_filters('the_content', $post->post_content),
         'number' => $episode->number,
-        'mnemonic' => $podcast->mnemonic.($episode->number < 100 ? '0' : '').($episode->number < 10 ? '0' : '').$episode->number
+        'mnemonic' => $podcast->mnemonic.($episode->number < 100 ? '0' : '').($episode->number < 10 ? '0' : '').$episode->number,
+        'soundbite_start' => $episode->soundbite_start,
+        'soundbite_duration' => $episode->soundbite_duration
         // @todo: all media files
     ]);
 }
 
+/**
+ * Check permission for change
+ */
+function update_episode_permission_check($request)
+{
+    if (!current_user_can( 'edit_posts')) {
+        return new WP_Error('rest_forbidden', 
+            esc_html__('sorry, you do not have permissions to use this REST API endpoint'),
+            array('status' => 401));
+    }
+    return true;
+}
+
+function episodes_update_api($request)
+{
+    $id = $request->get_param('id');
+    $episode = Episode::find_by_id($id);
+
+    if (!$episode)
+        return;
+
+    if ( isset($request['soundbite_start'])) {
+        $start = $request['soundbite_start'];
+        $episode->__set('soundbite_start', $start);
```

Comment:
`[__set](https://www.php.net/manual/en/language.oop5.overloading.php#object.set)` ist eine PHP Overloading Funktion, die nie direkt benutzt werden sollte. Du schreibst also:

```suggestion
        $episode->soundbite_start = $start;
```